stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -402,18 +504,13 @@ public class MessageListViewModel(
                 previousUser == user && nextUser != user -> Bottom
                 else -> None
             }
-
-            items.add(
-                MessageItem(
-                    message,
-                    position,
-                    parentMessageId,
-                    user.id == currentUser?.id
-                )
+            MessageItem(
+                message = message,
+                groupPosition = position,
+                parentMessageId = parentMessageId,
+                isMine = user.id == currentUser?.id
             )
-        }
-
-        return items
+        }.reversed()
```

Comment:
[Query] probably more curious why is it reversed ?

Reply:
We show message items in the reverse order, because we're reversing the message list too. This is the base way of showing a Chat, where the newest messages have to be at the bottom, rather than the top.